Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers_test.py

```diff
@@ -0,0 +1,117 @@
+
+
+def mock_client():
+    return Client(client_id='client_id',
+                  verify=False,
+                  proxy=False)
+
+
+def test_risky_users_list_command(requests_mock) -> None:
+    """
+    Scenario: List Risky Users.
+    Given:
+     - User has provided valid credentials.
+     - Headers and JWT token have been set.
+    When:
+     - risky_users_list_command is called.
+    Then:
+     - Ensure number of items is correct.
+     - Ensure outputs prefix is correct.
+     - Ensure outputs key fields is correct.
+    """
+    from AzureRiskyUsers import risky_users_list_command
+    mock_response = load_mock_response('list_risky_users.json')
+    requests_mock.post(ACCESS_TOKEN_REQUEST_URL, json={})
+    requests_mock.get(f'{BASE_URL}identityProtection/riskyUsers', json=mock_response)
+    result = risky_users_list_command(mock_client(), {'limit': '20', 'page': '1'})
+    assert result.outputs_prefix == 'AzureRiskyUsers.RiskyUser'
+    assert result.outputs_key_field == 'id'
+    assert len(result.raw_response) == 3
+
+
+def test_risky_user_get_command(requests_mock) -> None:
+    """
+    Scenario: Get Risky User.
+    Given:
+     - User has provided valid credentials.
+     - Headers and JWT token have been set.
+    When:
+     - risky_user_get_command is called.
+    Then:
+     - Ensure outputs prefix is correct.
+     - Ensure outputs key fields is correct.
+     - Ensure user ID is correct.
+
+    """
+    from AzureRiskyUsers import risky_user_get_command
+    mock_response = load_mock_response('get_risky_user.json')
+    requests_mock.post(ACCESS_TOKEN_REQUEST_URL, json={})
+    requests_mock.get(f'{BASE_URL}identityProtection/riskyUsers/1', json=mock_response)
+    result = risky_user_get_command(mock_client(), args={'id': '1'})
+    assert result.outputs_prefix == 'AzureRiskyUsers.RiskyUser'
+    assert result.outputs_key_field == 'id'
+    assert result.raw_response.get('id') == '1'
+
+
+def test_risk_detections_list_command(requests_mock) -> None:
+    """
+    Scenario: List Risk Detections.
+    Given:
+     - User has provided valid credentials.
+     - Headers and JWT token have been set.
+    When:
+     - risk_detections_list_command is called.
+    Then:
+     - Ensure outputs prefix is correct.
+     - Ensure outputs key fields is correct.
+     - Ensure risk detection ID is correct.
+
+    """
+    from AzureRiskyUsers import risk_detections_list_command
+    mock_response = load_mock_response('list_risk_detections.json')
+    requests_mock.post(ACCESS_TOKEN_REQUEST_URL, json={})
+    requests_mock.get(f'{BASE_URL}identityProtection/riskDetections', json=mock_response)
+    result = risk_detections_list_command(mock_client(), {'limit': '20', 'page': '1'})
+    assert result.outputs_prefix == 'AzureRiskyUsers.RiskDetection'
+    assert result.outputs_key_field == 'id'
+    assert result.raw_response.get('value')[0].get('id') == '1'
+
+
+def test_risk_detection_get_command(requests_mock) -> None:
```

Comment:
Please test pythonic logic and helper functions, not just the commands

Reply:
Done